changelogs/fragments/filetree_create_workflow_job_templates.yml

```diff
@@ -0,0 +1,4 @@
+---
+bugfixes:
+  - fix a truthy value
```

Comment:
Can we be more descriptive int he changelog so we know what we are actually talking about when we don't have the context of the PR